packages/artillery-plugin-publish-metrics/lib/open-telemetry/tracing/base.js

```diff
@@ -75,20 +75,21 @@ class OTelTraceConfig {
   }
 
   async shutDown() {
-    debug('Initiating TracerProvider shutdown');
+    this.debug('Initiating TracerProvider shutdown');
     try {
       await this.tracerProvider.shutdown();
     } catch (err) {
-      debug(err);
+      this.debug(err);
     }
-    debug('TracerProvider shutdown completed');
+    this.debug('TracerProvider shutdown completed');
   }
 }
 
 class OTelTraceBase {
   constructor(config, script) {
     this.config = config;
     this.script = script;
+    this.debug = require('debug')(`plugin:publish-metrics:${this.config.type}`);
```

Comment:
Setting `this.debug` here with reporter type will make it available to both HTTP and Playwright trace reporters.